src/__tests__/config/translations.test.ts

```diff
@@ -1,7 +1,45 @@
+import Path from 'path'
+import fs from 'fs'
 import translations from '../../config/localization/translations.json'
 
 describe('Check translations integrity', () => {
   it.each(Object.keys(translations))('Translation key value should be equal', (key) => {
     expect(key).toEqual(translations[key])
   })
 })
+
+describe('Check translations available', () => {
+  const files = []
+  const translationKeys = Object.keys(translations)
+
+  function throughDirectory(directory) {
+    fs.readdirSync(directory).forEach((file) => {
+      const absolute = Path.join(directory, file)
+      if (fs.statSync(absolute).isDirectory()) return throughDirectory(absolute)
+      if (absolute.includes('.tsx')) {
+        return files.push(absolute)
+      }
+      return files.length
+    })
+  }
+
+  throughDirectory('src/views')
+  throughDirectory('src/components')
```

Comment:
Is it faster to do it like this versus the entire `src` directory?

Reply:
Those are the 2 directories that we have translations, but I changed to "src" to traverse all directories